screens/discover/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { StyleSheet, Text, View } from 'react-native';
+import React, { useState } from 'react';
+import Header from '../home/header';
+import { LQDBottomSheet } from '@/components';
+
+const Discover = () => {
+  const [show, setShow] = useState(false);
+  return (
+    <View style={styles.container}>
+      <Header amount={3333} action={() => setShow((prev) => !prev)} />
+      <LQDBottomSheet show={show} title="Sort by" variant="primary" onClose={() => setShow((prev) => !prev)}></LQDBottomSheet>
```

Comment:
Instead of repeating these components in each tab screen, could we include it in the `_layout.tsx` file in `app/(tabs)/_layout.tsx`? That way, it'll automatically be available to all the children without the need to repeat it?

Reply:
@iyiolaosuagwu